src/Table/Table.spec.tsx

```diff
@@ -0,0 +1,68 @@
+// VENDOR
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { ThemeProvider } from '@xstyled/styled-components';
+import { shallow, mount } from 'enzyme';
+// ANCHOR
+import { RootTheme } from '../../src/theme';
+import { Table } from './Table.component';
+const { Head, Body, Cell, Footer, Row, HeaderCell } = Table;
+
+describe('Component: Table', () => {
+    it('should be defined.', () => {
+        const subject = (
+            <ThemeProvider theme={RootTheme}>
+                <Table />
+            </ThemeProvider>
+        );
+        const wrapper = mount(subject);
+        const component = shallow(subject);
+
+        expect(subject).toBeDefined();
+        expect(wrapper).toBeDefined();
+        expect(component).toBeDefined();
+        const tree = renderer.create(subject).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    it('should render with all the bells and whistles', () => {
```

Comment:
I don't love the kitchen sink approach here, but as these are mostly pass-throughs it seems ok.